Neos.ContentRepository.Core/Classes/Feature/WorkspaceCommandHandler.php

```diff
@@ -506,6 +510,7 @@ private function handlePublishIndividualNodesFromWorkspace(
         );
 
         $originalCommands = $this->extractCommandsFromContentStreamMetadata($workspaceContentStreamName);
+        $originalCommands = $this->compactCommands($originalCommands);
```

Comment:
Just a minor issue of course, but I would suggest to move the "feature flag" check out here, e.g.:
```suggestion
        if ($this->experiments->compactCommands_compressSimple()) {
            $originalCommands = $this->compactCommands($originalCommands);
        }
```